src/utils/generators/locales/en.json

```diff
@@ -0,0 +1,312 @@
+    "Business-focused",
+    "Centralized",
+    "Cloned",
+    "Compatible",
+    "Configurable",
+    "Cross-group",
+    "Cross-platform",
+    "Customer-focused",
+    "Customizable",
+    "Decentralized",
+    "De-engineered",
+    "Devolved",
+    "Digitized",
+    "Distributed",
+    "Diverse",
+    "Down-sized",
+    "Enhanced",
+    "Enterprise-wide",
+    "Ergonomic",
+    "Exclusive",
+    "Expanded",
+    "Extended",
+    "Face to face",
+    "Focused",
+    "Front-line",
+    "Fully-configurable",
+    "Function-based",
+    "Fundamental",
+    "Future-proofed",
+    "Grass-roots",
+    "Horizontal",
+    "Implemented",
+    "Innovative",
+    "Integrated",
+    "Intuitive",
+    "Inverse",
+    "Managed",
+    "Mandatory",
+    "Monitored",
+    "Multi-channelled",
+    "Multi-lateral",
+    "Multi-layered",
+    "Multi-tiered",
+    "Networked",
+    "Object-based",
+    "Open-architected",
+    "Open-source",
+    "Operative",
+    "Optimized",
+    "Optional",
+    "Organic",
+    "Organized",
+    "Persevering",
+    "Persistent",
+    "Phased",
+    "Polarised",
+    "Pre-emptive",
+    "Proactive",
+    "Profit-focused",
+    "Profound",
+    "Programmable",
+    "Progressive",
+    "Public-key",
+    "Quality-focused",
+    "Reactive",
+    "Realigned",
+    "Re-contextualized",
+    "Re-engineered",
+    "Reduced",
+    "Reverse-engineered",
+    "Right-sized",
+    "Robust",
+    "Seamless",
+    "Secured",
+    "Self-enabling",
+    "Sharable",
+    "Stand-alone",
+    "Streamlined",
+    "Switchable",
+    "Synchronised",
+    "Synergistic",
+    "Synergized",
+    "Team-oriented",
+    "Total",
+    "Triple-buffered",
+    "Universal",
+    "Up-sized",
+    "Upgradable",
+    "User-centric",
+    "User-friendly",
+    "Versatile",
+    "Virtual",
+    "Visionary",
+    "Vision-oriented",
+    "Small",
+    "Ergonomic",
+    "Rustic",
+    "Intelligent",
+    "Gorgeous",
+    "Incredible",
+    "Fantastic",
+    "Practical",
+    "Sleek",
+    "Awesome",
+    "Generic",
+    "Handcrafted",
+    "Handmade",
+    "Licensed",
+    "Refined",
+    "Unbranded",
+    "Tasty",
+    "Red",
+    "Green",
+    "Blue",
+    "Yellow",
+    "Purple",
+    "Mint green",
+    "Teal",
+    "White",
+    "Black",
+    "Orange",
+    "Ivory",
+    "Indigo",
+    "Gold",
+    "Fuchsia",
+    "Cyan",
+    "Azure",
+    "Lavender",
+    "Silver",
+    "Steel",
+    "Wooden",
+    "Concrete",
+    "Plastic",
+    "Cotton",
+    "Granite",
+    "Rubber",
+    "Metal",
+    "Soft",
+    "Fresh",
+    "Frozen"
+  ],
+  "noun": [
+    "giant panda",
+    "spectacled bear",
+    "sun bear",
+    "sloth bear",
+    "american black bear",
+    "asian black bear",
+    "brown bear",
+    "polar bear",
+    "bulldog",
+    "bullmastiff",
+    "bully kutta",
+    "burgos pointer",
+    "affenpinscher",
+    "afghan hound",
+    "aidi",
+    "airedale terrier",
+    "akbash",
+    "akita",
+    "alano español",
+    "alapaha blue blood bulldog",
```

Comment:
Some of those get a bit too long 
![obraz](https://user-images.githubusercontent.com/2972266/111353658-cc391f00-8685-11eb-8fac-747fa15d77d3.png)
I'd stick to simple, short words